jscomp/bsb/templates/basic/package.json

```diff
@@ -2,16 +2,11 @@
   "name": "${bsb:name}",
   "version": "${bsb:proj-version}",
   "scripts": {
-    "clean": "bsb -clean-world",
     "build": "bsb -make-world",
-    "watch": "bsb -make-world -w"
+    "start": "bsb -make-world -w",
```

Comment:
Mmmh you might want to change this to be `./lib/bs/bytecode/index.byte` so that it's more of a "start".
You could keep the "watch" for the `-w`.